.github/workflows/run-style-checks.yml

```diff
@@ -0,0 +1,28 @@
+name: Run style checks on codebase
+
+on:
+  pull_request
+
+jobs:
+  build:
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+        python-version: [3.6, 3.8]
```

Comment:
Not sure why we are running the style checks on both 3.6 and 3.8.

Reply:
Just for safety: I want to include both the main Python version that Enthought currently cares about, and a recent Python version. Same story for the documentation build.

Reply:
To take one motivating example: we want to be notified about pieces of syntax that become unsupported or deprecated in future versions of Python. For example, flake8 running on Python 3.5 won't warn about deprecated string literals like `"\s"`, while flake8 running on Python 3.8 will.

Reply:
Hmm. I take it back. Looks like flake8 on Python 3.5 does give the same warnings. I guess it's conceivable that flake8 gives exactly the same results regardless of Python version. I don't think I'm willing to rely on that, though, unless it's clearly documented to be the case.

Reply:
> unless it's clearly documented to be the case

Ah, quite the opposite: at https://flake8.pycqa.org/en/latest/, they say:

> In many ways, Flake8 is tied to the version of Python on which it runs.

So there _is_ scope for different results with different Python versions, and so I think it makes sense to keep the style checks on at least two different Python versions.